processor/src/bitwise/mod.rs

```diff
@@ -300,8 +224,6 @@ impl Bitwise {
     /// - Columns 8 to 11 are set to the 4 least-significant bits of `b`.
     /// - Column 12 and 13 are left for the output value and that of the previous row, which are
     ///   set elsewhere.
-    /// - Column 14 is padded with 0 so the width of the bitwise trace row will equal the width of
-    ///   power of two operation trace rows.
```

Comment:
The top of this comment block needs to be updated as well from "populates the first 15 columns of trace as follows"

Reply:
Ah yes! done.